vardbg/assets/styles/wood.py

```diff
@@ -0,0 +1,18 @@
+scheme = {
+    'base00' : '#231e18',
+    'base01' : '#302b25',
+    'base02' : '#48413a',
+    'base03' : '#9d8b70',
+    'base04' : '#b4a490',
+    'base05' : '#cabcb1',
+    'base06' : '#d7c8bc',
+    'base07' : '#e4d4c8',
+    'base08' : '#d35c5c',
+    'base09' : '#ca7f32',
+    'base0a' : '#e0ac16',
+    'base0b' : '#b7ba53',
+    'base0c' : '#6eb958',
+    'base0d' : '#88a4d3',
+    'base0e' : '#bb90e2',
+    'base0f' : '#b49368'
```

Comment:
This does not match Black's code style, which is what this project uses. Please run the pre-commit hooks to format it.